packages/hub/src/hub.ts

```diff
@@ -35,7 +35,7 @@ import { Session } from './session';
  *
  * @hidden
  */
-export const API_VERSION = 3;
+export const API_VERSION = 4;
```

Comment:
What's this?

Reply:
This is basically our safeguard for making sure if multiple `@sentry/hub` packages exist in node_modules, we always load the latest one.